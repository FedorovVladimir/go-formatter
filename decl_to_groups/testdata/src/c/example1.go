package main

var ( // want "incorrect single declaration style"
	v1 = "v1"
)
var v2 = "v2"

var v3 = "v3" // want "incorrect single declaration style"
var v4 = "v4"

var v5 = "v5" // want "incorrect single declaration style"
var (
	v6 = "v6"
)
