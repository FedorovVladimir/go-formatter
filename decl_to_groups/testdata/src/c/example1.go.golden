package main

var (
	// want "incorrect declaration style"
	v1 = "v1"
	v2 = "v2"
)

var (
	v3 = "v3" // want "incorrect declaration style"
	v4 = "v4"
)

var (
	v5 = "v5" // want "incorrect declaration style"
	v6 = "v6"
)

const (
	c1 = "c1" // want "incorrect declaration style"
)

var (
	v7 = "v7" // want "incorrect declaration style"
)